src/SFA.DAS.AssessorService.Infrastructure/ApiClients/OuterApi/ApiClient.cs

```diff
@@ -18,6 +18,7 @@ public OuterApiClient(HttpClient httpClient, IOuterApiClientConfiguration config
         {
             _httpClient = httpClient;
             _configuration = configuration;
+            _httpClient.BaseAddress = new Uri(_configuration.BaseUrl);
```

Comment:
Where has this come from?

Reply:
I added this in when I was trying to diagnose the configuration issues. I forgot to remove it afterwards. Corrected in latest commit.